### PR TITLE
Upgrade EVS to 1.1

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

**Comment by rshym** [comment]
Includes commit from #4

File: apps/default/Android.bp [hunk 1]
```diff
@@ -58,14 +58,14 @@ cc_binary {
     ],
 
     required: [
-        "config.json",
-        "CarFromTop.png",
-        "LabeledChecker.png",
+        "config.xt.json",
+        //"CarFromTop.png",
+        //"LabeledChecker.png",
```
Comment (arminn) [inline apps/default/Android.bp]: Do we need those comments here?

File: apps/default/Android.bp [hunk 2]
```diff
@@ -78,25 +78,25 @@ cc_binary {
...
-
-    src: "LabeledChecker.png",
-    sub_dir: "automotive/evs",
+//prebuilt_etc {
+//    name: "CarFromTop.png",
+//    vendor: true,
+//    src: "CarFromTop.png",
+//    sub_dir: "automotive/evs",
+//
+//}
```
Comment (arminn) [inline apps/default/Android.bp]: Do we need those comments here?

File: apps/default/Android.bp [hunk 3]
```diff
@@ -78,25 +78,25 @@ cc_binary {
...
 
-}
+//prebuilt_etc {
+//    name: "LabeledChecker.png",
+//    vendor: true,
+//    src: "LabeledChecker.png",
+//    sub_dir: "automotive/evs",
+//
+//}
```
Comment (arminn) [inline apps/default/Android.bp]: Do we need those comments here?

File: sampleDriver/Android.bp [hunk 4]
```diff
@@ -110,16 +109,16 @@ cc_library{
...
-    soc_specific: true,
-    src: "resources/evs_configuration.dtd",
-    sub_dir: "automotive/evs",
-}
+//prebuilt_etc {
+//    name: "evs_configuration.dtd",
+//    soc_specific: true,
+//    src: "resources/evs_configuration.dtd",
+//    sub_dir: "automotive/evs",
+//}
```
Comment (arminn) [inline sampleDriver/Android.bp]: Do we need those comments here?

**Review by arminn — CHANGES_REQUESTED** [review]

File: apps/default/Android.bp (hunk 1 above)
Comment (rshym) [reply apps/default/Android.bp]: Commented lines are removed.

File: apps/default/Android.bp (hunk 2 above)
Comment (rshym) [reply apps/default/Android.bp]: Commented lines are removed.

File: apps/default/Android.bp (hunk 3 above)
Comment (rshym) [reply apps/default/Android.bp]: Commented lines are removed.

File: sampleDriver/Android.bp (hunk 4 above)
Comment (rshym) [reply sampleDriver/Android.bp]: Commented lines are removed.

**Pull request merged**